src/main/webapp/src/app/hidden-gem.service.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { HiddenGemService } from "./hidden-gem.service";
+import { HiddenGem } from './hidden-gem';
+import { HttpErrorResponse } from '@angular/common/http';
+import { of } from 'rxjs';
+import { fakeAsync } from '@angular/core/testing';
+
+let httpClientSpy: { get: jasmine.Spy };
+let hiddenGemService: HiddenGemService;
+
+beforeEach(() => {
```

Comment:
Can you put all this inside a describe function? that way tests that logically belong together are grouped and it's easy to find them in the jasmine html display